lib/src/components/NxTextInput/NxTextInput.scss

```diff
@@ -6,17 +6,18 @@
  */
 @import '../../scss-shared/nx-variables';
 @import '../../scss-shared/_nx-form-validation.scss';
+@import '../../scss-shared/_nx-container-helpers.scss';
 
-.nx-text-input:not(:disabled) {
-  &.nx-text-input--valid {
+.nx-text-input {
+  &.valid:not(.pristine) {
```

Comment:
This `:not` shouldn't be necessary, depending on how the `pristine` mixin is defined

Reply:
> This `:not` shouldn't be necessary, depending on how the `pristine` mixin is defined

https://github.com/sonatype/sonatype-react-shared-components/pull/57/commits/600837d63c526088a976a7de4384b2426cb07531